.github/ISSUE_TEMPLATE/EDITOR_ISSUE_TEMPLATE.yml

```
@@ -0,0 +1,39 @@
+name: "\U0001F50D Editor's Issue Template"
+description: Template to propose editing a chapter
+title: "[TO EDIT] <chapter/subchapter>"
+body:
+- type: markdown
+  attributes:
+    value: "This template will guide you through the steps necessary for you to propose a chapter/subchapter to edit, review or restructure. Please fill in the title above but do not remove the text “[TO EDIT]”. If you have the required permissions, please assign the appropriate label (or labels) to your issue; if you do not have the permissions, one of the project members can help with this after you create the issue."
+- type: textarea
+  id: summary
+  attributes:
+    label: Summary
+    description: "Please provide a detailed description of the chapter/subchapter you propose to edit, review or restructure. Please provide as much context as possible and link to related issues and/or pull requests. Providing links to sections you're referencing for clarity is highly encouraged."
+  validations:
+    required: true
+- type: textarea
+  id: todo
+  attributes:
+    label: "What needs to be done?"
+    description: "Please comment on the overall structure of the file. Your response can cover the questions such as, “Do sections need clearer headings? Are there any technical terms that might not be understood? Is something missing that is worth discussing before adding?” We suggest using bullets (indicated by * or -) and filled checkboxes ([x]) here."
+    placeholder: Please use bullets (* or -) and filled checkboxes ([x]) as necessary.
```

```suggestion
    description: "Please comment on the overall structure of the file. Your response can cover the questions such as, “Do sections need clearer headings? Are there any technical terms that might not be understood? Is something missing that is worth discussing before adding?”"
    placeholder: Please use bullets (* or -) and filled checkboxes ([x]) as necessary.
```

My opinion is that it doesn't add much value to repeat this in both the description and the placeholder